gdal/doc/gdal_tutorial.dox

```diff
@@ -417,15 +420,15 @@ In C:
 In Python:
 
 \code
-    format = "GTiff"
-    driver = gdal.GetDriverByName( format )
+    fileformat = "GTiff"
+    driver = gdal.GetDriverByName(fileformat)
     metadata = driver.GetMetadata()
-    if metadata.has_key(gdal.DCAP_CREATE) \
-       and metadata[gdal.DCAP_CREATE] == 'YES':
-        print 'Driver %s supports Create() method.' % format
-    if metadata.has_key(gdal.DCAP_CREATECOPY) \
-       and metadata[gdal.DCAP_CREATECOPY] == 'YES':
-        print 'Driver %s supports CreateCopy() method.' % format
+
+    if metadata.get(gdal.DCAP_CREATE) == "YES":
```

Comment:
Using dict.get(key) returns either the value or None (~=False), this makes the condition much simpler.